boot/freeldr/freeldr/arch/i386/pcmem.c

```diff
@@ -350,7 +355,7 @@ PcMemGetBiosMemoryMap(PFREELDR_MEMORY_DESCRIPTOR MemoryMap, ULONG MaxMemoryMapSi
 
         if (PcBiosMemoryMap[PcBiosMapCount].Length == 0)
         {
-            TRACE("Discard empty range. (would-be-PcBiosMapCount = %lu, BaseAddress = %lu, Length = 0)\n",
+            TRACE("Discard empty range. (would-be-PcBiosMapCount = %lu, BaseAddress = 0x%llx, Length = 0)\n",
```

Comment:
Is this change needed to print pointers (of type PVOID) ? If so, why not using 0x%p ?

Reply:
BaseAddress is ULONGLONG so this is right.